server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/CobolLexer.g4

```diff
@@ -76,6 +76,7 @@ SKIP2 : S K I P '2' DOT_FS? -> skip;
 SKIP3 : S K I P '3' DOT_FS?-> skip;
 
 // keywords
+ABORT : A B O R T;
```

Comment:
Plz. check duplication for all tokens, same here, duplicated, already exist in other lexer